packages/babel-plugin-transform-destructuring/test/fixtures/destructuring/empty-string/exec.js

```diff
@@ -0,0 +1,3 @@
+let [a] = '';
+
+expect(undefined).toBe(a)
```

Comment:
Nit:
```suggestion
expect(a).toBe(undefined)
```

Usually the "known" value should be put in the `toBe` clause, while the "value to be tested" in `expect()`.

Reply:
Ooops. I'll fix it.